server/utils/referralUtils.test.ts

```diff
@@ -124,5 +124,14 @@ describe('ReferralUtils', () => {
         text: 'completed',
       })
     })
+
+    it('updates check answers task when the the referral is ready for submission', () => {
+      const referralWithOasysConfirmed = referralFactory.build({ oasysConfirmed: true })
+      const taskListSections = ReferralUtils.taskListSections(referralWithOasysConfirmed)
+      const checkAnswersSection = taskListSections[2].items[0]
```

Comment:
Is there a way we can make this more resistant to a possible future change to the order of sections, rather than using array indexes?

Reply:
Maybe using the new `testIds` method?

Reply:
Just realised this isn't an integration test. I'm not really sure what we could do within this context. Turning it into an object will make it less easy to use in the view

Does it need to be resistant? Would an unexpected failing test not just prompt us to review and revise this if the order changes?

Reply:
We could - technically less performant, but since the array is and will remain short, I don't think that's an issue. Will update

Reply:
(Of course this makes it less resistant to changes to the heading, inversely 😅)